### ISSUE TITLE
Support for Tags and Project Description

### ISSUE THREAD

**Issue by RobSHK** [Zargath/azure-pipelines-dependency-track#10]
Hello,

It would be great if a support for project Properties which includes Project Tags and Project description will be implemented.

TIA

**Comment by Zargath** [comment]
Hi @RobSHK,

Thanks for opening an issue for this. I've started trying things out for this. I think the best option would to have a second task for this kind of thing. It could get big pretty quickly and I could see situations where someone might want to update a project without uploading a BOM.

Stay tuned!

**Comment by Zargath** [comment]
Hi @RobSHK,

That's an option as well. Maybe if we limit it to Tags and description for the upload bom task. Another task could exist for updating the project's other properties. I'm just don't want the upload task to creep into a task with too many responsibilities. Keeping the responsibilities separate allows for more flexibility.  

Thanks!

**Comment by RobSHK** [comment]
Yes, that would also work. Is there an estimate for the implementation of this solution?

**Comment by RobSHK** [comment]
Hi @Zargath, 
Could you estimate when the additional features will be released?

TIA!

**Comment by Zargath** [comment]
Hey @RobSHK,
The code in the Pull Request works, but I haven't had enough time to test it fully. When I've been able to test it properly, I will release it.

Thanks!
